### PR TITLE
[test] Delete `"@org_tensorflow//tensorflow/core:lib",` and test

### PR DESCRIPTION
**Pull request by ManfeiBai**

### CODE REVIEW

File: third_party/xla_client/BUILD [hunk 1]
```diff
@@ -193,7 +194,6 @@ cc_library(
     "@org_tensorflow//tensorflow/compiler/xrt/cc:xrt_ops",
     "@org_tensorflow//tensorflow/core:core_cpu",
     "@org_tensorflow//tensorflow/core:framework_internal",
-    "@org_tensorflow//tensorflow/core:lib",
```
Comment (will-cromar) [inline third_party/xla_client/BUILD:196]: You don't need to delete this reference or the one under `xrt_local_service`. These are only used by XRT.

Comment (ManfeiBai) [reply third_party/xla_client/BUILD]: Thanks, Will, updated

**Comment by ManfeiBai** [comment]
only delete core:lib in BUILD and will update `torch/csrc/BUILD` later

**Comment by ManfeiBai** [comment]
will close this PR since all the deps has been treated in seperated PR:
https://github.com/pytorch/xla/pull/5174
https://github.com/pytorch/xla/pull/5175
https://github.com/pytorch/xla/pull/5176

**Pull request closed without merging**